msteams-platform/bots/how-to/Teams conversational AI/bots-and-tabs.md

```diff
@@ -0,0 +1,311 @@
+---
+title: Synchronized bots and tabs
+description: In this article, learn about interoperability between bot and tab.
+ms.localizationpriority: medium
+ms.topic: coceptual
+ms.author: v-npaladugu
+author: surbhigupta
+ms.date: 05/02/2024
+---
+
+# Synchronized bots and tabs
+
+Bots and tabs are two capabilities of Microsoft Teams that can be used together to create a more engaging and interactive user experience. Bots are great way for conversational experiences through chat, while tabs allow you to embed web applications directly into Teams. Tabs and bots can be combined to create more customized interactions between the user and an app, such as modifying a collaborative document or filling out a web-based form.
```

Comment:
> Bots are great way for conversational experiences through chat
Grammar

Reply:
Please review the text in the doc for flow and grammar. There are places where it reads awkwardly, for example because there are missing words/articles.

Reply:
Updated